README.adoc

```diff
@@ -30,28 +30,25 @@ The Planner build can be quickly run using the link:scripts/run-planner.sh[launc
 
 You can use the launcher script to run the Planner either as a Standalone component or as an Integrated component using an external webapp like the fabric8-ui.
 
-- To run a minimal runtime implementation of the Planner using an in-memory mock database use:
+. *Without Docker:* To run a minimal runtime implementation of the Planner using an in-memory mock database use:
```

Comment:
This should be a Bulleted list since they are not sequential steps. 

The headings 'with Docker' and 'without Docker' do not seem necessary here since the immediate next sentence makes it clear. Standalone/Minimal runtime and Integrated Application seem more apt as titles, i.e. if really required.
 
Also, Docker Compose should be in back ticks like gulp below.